sherpa/utils/testing.py

```diff
@@ -141,23 +143,16 @@ def decorator(test_function):
 
         return decorator
 
-    def requires_plotting(test_function):
-        """
-        Decorator for test functions requiring a plotting library.
-        """
-        packages = ('pylab', )
-        msg = "plotting backend required"
-        return requires_package(msg, *packages)(test_function)
 
     def requires_pylab(test_function):
         """
-        Returns True if the pylab module is available (pylab).
+        Set plotting backend to PylabBackend if available.
         Used to skip tests requiring matplotlib
```

Comment:
I know this text is not from this PR, but I think we got the sense of it wrong. Given that the test has slightly changed, perhaps it should just be something like

```
Runs the test with the PylabBackend plotting backend if available, otherwise skips the test.
```